docs/source/tutorials/one_class_segmentation_2d_unet.rst

```diff
@@ -24,137 +27,345 @@ Download dataset
+
+                .. code-block:: json
+
+                    "slice_axis": "axial"
 
     - ``loader_parameters:multichannel``: Turn on/off multi-channel training. If ``true``, each sample has several
       channels, where each channel is an image contrast. If ``false``, only one image contrast is used per sample.
 
-      .. code-block:: json
+        .. tabs::
+
+            .. tab:: JSON File
+
+                Usually around `this line: <https://github.com/ivadomed/ivadomed/blob/master/ivadomed/config/config.json#L30>`__ in the ``config.json`` is where you can update the ``multichannel`` subkey within the ``loader_parameters`` sub-dictionary.
+
+                .. code-block:: json
 
-         "multichannel": false
+                    "multichannel": false
 
-      .. note::
+                .. note::
 
-         The multichannel approach requires that for each subject, the image contrasts are co-registered. This implies that
-         a ground truth segmentation is aligned with all contrasts, for a given subject. In this tutorial, only one channel
-         will be used.
+                    The multichannel approach requires that for each subject, the image contrasts are co-registered. This implies that
+                    a ground truth segmentation is aligned with all contrasts, for a given subject. In this tutorial, only one channel
+                    will be used.
 
     - ``training_time:num_epochs``: the maximum number of epochs that will be run during training. Each epoch is composed
       of a training part and an evaluation part. It should be a strictly positive integer.
 
-      .. code-block:: json
+        .. tabs::
 
-         "num_epochs": 100
+            .. tab:: JSON File
+
+                Usually around `this line: <https://github.com/ivadomed/ivadomed/blob/master/ivadomed/config/config.json#L29>`__ in the ``config.json`` is where you can update the ``slice_axis`` subkey within the ``loader_parameters`` sub-dictionary.
+
+                .. code-block:: json
+
+                    "num_epochs": 100
 
 Train model
 -----------
 
     Once the configuration file is ready, run the training:
 
-    .. code-block:: bash
+    .. tabs::
 
-       ivadomed --train -c config.json --path-data path/to/bids/data --path-output path/to/output/directory
+        .. tab:: Command Line Interface
 
-    - We can pass other flags to execute different commands (training, testing, segmentation), see `Usage <../usage.html>`__.
+            .. code-block:: bash
 
+               ivadomed --train -c config.json --path-data path/to/bids/data --path-output path/to/output/directory
 
-    - ``--path-output``: Folder name that will contain the output files (e.g., trained model, predictions, results).
+    - In the above command, we execute the ``train`` command and manually specified ``path-data`` and ``path-output`` and overwrote/replace the specification in ``config.json``
```

Comment:
```suggestion
    - In the above command, we execute the ``--train`` command and manually specified ``--path-data`` and ``--path-output`` and overwrote/replace the specification in ``config.json``
```
The "--" would help to distinguish between the command and the config field (ex: path-data with a dash and path_data with an underscore).

Reply:
Good point. I concur.